CPS-0004/README.md

```diff
@@ -0,0 +1,96 @@
+
+To frame this discussion, it's important to outline a set of goals and baseline truths. Some of these will be neutral and uncontroversial, while others are in need of community discussion.
+
+### **Acknowledgements**
+<details>
+<summary> Thank you to the following people who helped review this draft before it was published: </summary>
+
+ * Adam K. Dean
+ * Jared Corduan
+ * Venessa Harris
+ * HeptaSean
+
+</details>
+
+## **Problem**
+
+At the core of it, a system of governance boils down to how decisions are made and enforced.  Cardano currently has 3 core decisions that greatly impact the operation of the chain:
+
+ - When and how to change the ledger rules (a "Controlled Hard Fork")
+   - These changes are often drastic, with fundamental tradeoffs and extremely high risks
+ - When and how to update various protocol parameters
+   - These parameter changes, while limited in scope compared to a hard fork, can still have potentially existential risks on the chain
+ - When and to whom to disburse funds from the treasury
+   - Over the last several years, this treasury has accumulated nearly 1 billion ADA, currently worth hundreds of millions of dollars
+
+The impact of these decisions are also often closely intertwined: development of the next hard fork may need funding from the treasury; parameters may need to be tweaked after a hard fork; etc.
+
+Today, these decisions are primarily made by the founding entities (IOG, Cardano Foundation, and Emurgo). While these entities often consult the community when making these decisions, the final power lies in their hands, with very little that can be done beyond an uncontrolled hard fork by the stake pool operators.
+
+For a variety of reasons, including moral, philosophical, financial, and possibly regulatory, these founding entities want to dilute their own power substantially, and share the burden of governance of the chain with the community that has arisen around it.
+
+IOG in particular has continued to provide skill and resources for years longer than initially intended, and is significantly over-budget and over-extended. IOG has said that they may be unable or unwilling to support development of the chain indefinitely, and so an expedient and iterative approach to governance is likely to have higher positive impact on our ability to maintain velocity as an ecosystem.
+
+## **Goals**
+
+Synthesizing from the above, I posit these (hopefully uncontroversial) goals, constraints, considerations, or ground truths for any governance system. I'm only one person, though, so this list will very likely be revised and updated by collective community input as discussion unfolds.
+
+1. A system of on-chain rules should allow decisions to be made and enacted regarding (at a minimum) the three types of changes listed above.
+2. Every (or nearly every) ADA holder should be able to meaningfully participate in this decision making process.
```

Comment:
To the 'nearly every' comment, which ADA holders are not able to participate?

Reply:
This is to leave space for solutions that have anti-DDOS measures like a small deposit; if you have 1 ada you can't pay the deposit, or sometimes even the transaction fees.